src/ResultsHouseblock/ResultsHouseblock.js

```diff
@@ -168,19 +184,31 @@ class ResultsHouseblock extends Component {
       </div>
     )
 
-    const printButton = isWithinNextTwoWorkingDays(date) && (
-      <PrintButton
-        id="printButton"
-        className="margin-left margin-top pull-right"
-        onClick={e => {
-          if (e) e.preventDefault()
-          handlePrint()
-        }}
-      >
-        <img className="print-icon" src="/images/Printer_icon_white.png" height="23" width="20" alt="Print icon" />
-        Print list
-      </PrintButton>
+    const printButton = (
+      <div id="buttons" className="buttons">
+        {isWithinNextTwoWorkingDays(date) && (
+          <PrintButton
+            id="printButton"
+            className=""
+            onClick={e => {
+              if (e) e.preventDefault()
```

Comment:
Is the `preventDefault` needed?

Reply:
Is preventDefault needed here?

Reply:
As per convo - added `type=button`.  That's fixed it.  Removed the preventDefault which was preventing the previous implementation from submitting a form.